fhir-path/src/main/java/com/ibm/fhir/path/util/FHIRPathUtil.java

```diff
@@ -415,6 +588,29 @@ public static TemporalAmount getTemporalAmount(FHIRPathQuantityValue quantityVal
         }
     }
 
+    public static ChronoUnit getChronoUnit(String unit) {
+        switch (unit) {
+        case "years":
+            return ChronoUnit.YEARS;
+        case "months":
+            return ChronoUnit.MONTHS;
+        case "weeks":
+            return ChronoUnit.WEEKS;
+        case "days":
+            return ChronoUnit.DAYS;
+        case "hours":
+            return ChronoUnit.HOURS;
+        case "minutes":
+            return ChronoUnit.MINUTES;
+        case "seconds":
+            return ChronoUnit.SECONDS;
+        case "milliseconds":
+            return ChronoUnit.MILLIS;
+        default:
+            throw new IllegalArgumentException("Unsupported unit: " + unit);
+        }
+    }
```

Comment:
should it support variations of those string units per the fhirpath spec

Reply:
I wanted to simplify it and only allow the equivalent of what the underlying java.time library allows